scripts/upgrade

```diff
@@ -225,22 +93,23 @@ else
 fi
 
 # Upgrade Piwigo via cURL
-ynh_local_curl "/upgrade.php?language=$applanguage&now=true" "language=$applanguage" "username=$admin" "password=$admin_pwd"
+# FIXME : Why Curl ... Why can't we trigger the migration from the command line somehow ...
+#ynh_local_curl "/upgrade.php?language=$applanguage&now=true" "language=$applanguage" "username=$admin" "password=$admin_pwd"
```

Comment:
So tests are somewhat passing but still need to fix this .. looking at the code this doesnt seem really possible to do this from command line >_>

Reply:
What's the issue ? ynh_local_curl doesn't work anymore ?